app/services/discovery_engine/evaluation/evaluation_runner.rb

```diff
@@ -52,9 +52,10 @@ def sample_set_name
 
       def fetch_with_wait
         while (e = fetch_evaluation)
+          Rails.logger.info("Still waiting for evaluation to complete...")
```

Comment:
I think that should probably stay where it is, as we wouldn't want to print it in the successful case